### DISCUSSION TITLE
安装alist后查看alist状态为failed

### DISCUSSION THREAD

**Discussion by RAINighty** [AlistGo/alist#3253] [Q&A]
通过一键安装脚本安装后使用systemctl status alist查看，提示`Active:failed(Result:signal)`
环境：hyper-v虚拟机ubuntu22.04
`dong@Dong:~/storage$ systemctl status alist`
`× alist.service - Alist service`
`     Loaded: loaded (/etc/systemd/system/alist.service; enabled; vendor preset: enabled)`
`     Active: failed (Result: signal) since Sat 2023-02-04 00:23:29 CST; 1s ago`
`    Process: 980985 ExecStart=/opt/alist/alist server (code=killed, signal=SEGV)`
`   Main PID: 980985 (code=killed, signal=SEGV)`
`        CPU: 536us`
`2月 04 00:23:29 Dong systemd[1]: Started Alist service.`
`2月 04 00:23:29 Dong systemd[1]: alist.service: Main process exited, code=killed, status=11/SEGV`
`2月 04 00:23:29 Dong systemd[1]: alist.service: Failed with result 'signal'.`

**Comment by RAINighty** [comment]
去/opt/alist运行./alist时说`段错误（核心已转储）`

**Comment by angelrand** [comment]
我也遇到同样的问题，fedora37
× alist.service - Alist service
     Loaded: loaded (/etc/systemd/system/alist.service; enabled; preset: disabl>
     Active: failed (Result: signal) since Tue 2023-02-07 15:33:03 CST; 12min a>
   Duration: 11ms
    Process: 953 ExecStart=/opt/alist/alist server (code=killed, signal=SEGV)
   Main PID: 953 (code=killed, signal=SEGV)
        CPU: 1ms

2月 07 15:33:03 fedora systemd[1]: Started alist.service - Alist service.
2月 07 15:33:03 fedora systemd[1]: alist.service: Main process exited, code=kil>
2月 07 15:33:03 fedora systemd[1]: alist.service: Failed with result 'signal'.
[root@fedora alist]# ./alist
段错误
[root@fedora alist]# ./alist admin
段错误
[root@fedora alist]#

**Comment by Mestmeck** [comment]
我这边是因为宝塔开启了宝塔系统加固，将宝塔系统加固关闭即可，或者把alist加入到异常进程监控的白名单里面
